### DISCUSSION TITLE
Manage Translations

### DISCUSSION THREAD

**Discussion by Amtech281** [firecmsco/firecms#406] [General]
hi @fgatti675 

We are using several langages as you do also in your demo project. Having a project with several languages needs careful management of translations. For example our master language is "german", and whenever we update a german text, we must ensure that we translate it to english.

How are you managing this in your project? Ideally we should be able to track update-time per field (not document) and then retrieve all documents, where text in the german field changed. Is this possible?
Have you done this before with firecms, you have some recommendations?

thanks, and have a great day,
Andre

**Comment by fgatti675** [comment]
Hi @Andre3424 Sorry for the late reply.
If you need such a fine control, I would create a backend function that makes a copy of the document every time there is a change, and saves it in a subcollection.
Then you could show that subcollection in the CMS

**Reply by Amtech281** [reply]
@fgatti675 just saw now your reply. Thanks, this is a good idea :-)
